nil/services/relayer/Makefile.inc

```
@@ -0,0 +1,18 @@
+root_services = nil/services
+root_relayer = nil/services/relayer
+
+.PHONY: $(root_relayer)/embed_l2_abi
+$(root_relayer)/embed_l2_abi:
+	@cat $(root_services)/rollup-bridge-contracts-compiled/contracts/bridge/l2/L2BridgeMessenger.sol/L2BridgeMessenger.json | jq .abi > $(root_relayer)/internal/l2/L2BridgeMessenger.json.abi
```

Imho, `.json` is a primary extension
```suggestion
	@cat $(root_services)/rollup-bridge-contracts-compiled/contracts/bridge/l2/L2BridgeMessenger.sol/L2BridgeMessenger.json | jq .abi > $(root_relayer)/internal/l2/L2BridgeMessenger.abi.json
```

---

It was done to be aligned with current gitignore rules (they ignore .abi). I can add narrow rule for this file here but not sure if it is worth it